src/Component/Atom/SquareImage.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import styled from 'styled-components';
+
+type SquareImageprops = {
+    id : string;
+}
+
+const SquareImage = ( {id} : SquareImageprops ) => {
+    return <Image src = {`../Asset/Image/${id}.png`}/>
```

Comment:
이 또한 저번주 코드리뷰에 남겨 드렸던 내용인데, 경로 설정도 잘못되어 있어서 아래와같이 수정하시면 해결됩니다!
```suggestion
    return <Image src={require(`../../Asset/Image/${id}.png`)} />;
```